File: Year 2/Semester 1/Object Oriented Programming/Labs/Labs/Book.java

public class Book {
	private String title,author;
	private int isbn;
	private double price;
	
	public Book(String title, int isbn, String author, double price){
		this.title = title;
		this.isbn = isbn;
		this.author = author;
		this.price = price;
	}
	public String getTitle(){
		return title;
	}
	public int getIsbn(){
		return isbn;
	}
	public String getAuthor(){
		return author;
	}
	public double getPrice(){
		return price;
	}
	public String toString(){
		return "\n----Book Details----\nTitle: " + title +"\nISBN: " + isbn +"\nAuthor: " + author +"\nPrice: " + price;
	}
}

File: Year 3/Semester 1/project/Files/Examples/mvc/model/Book.php
<?php

class Book {
	public $title;
	public $author;
	public $description;
	
	public function __construct($title, $author, $description)  
    {  
        $this->title = $title;
	    $this->author = $author;
	    $this->description = $description;
    } 
}

?>